src/trace-agent.js

```diff
@@ -66,7 +66,7 @@ function TraceAgent(config, logger) {
 TraceAgent.prototype.stop = function() {
   hooks.deactivate();
   pluginLoader.deactivate();
-  cls.destroyNamespace();
+  this.policy = new tracingPolicy.TraceNonePolicy();
```

Comment:
Add a comment here explaining why this is necessary – namely that unpatching may not necessarily completely remove trace generation from instantiated objects.